### PR TITLE
Updated backend-tasks readme

### PR DESCRIPTION
**Pull request by awanlin**
Signed-off-by: Andre Wanlin <awanlin@rapidrtc.com>

## Hey, I just made a Pull Request!

Updated the backend-tasks README based on usage:

- It did not like dashes in the `id`, changed that to an underscore
- The `timeout` is required, added that to the example
- Added a note about tasks not running as expected for local development using persistent database

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by awanlin** [comment]
@freben some changes based on my usage. Not sure if the `timeout` example is clear, happy to change

File: packages/backend-tasks/README.md [hunk 1]
```diff
@@ -21,14 +21,19 @@ import { Duration } from 'luxon';
...
+  timeout: Duration.fromObject({ minutes: 15 }),
   fn: async () => {
     await entityProvider.run();
   },
 });
 ```
 
+## Local Development
+
+When working with backend-tasks you may run into your task not running as expected if you are using a persistent database. A table called "backstage_backend_tasks\_\_tasks" in the applicable plugin's database will contain a record with the next run date and time. You can delete this record to get things back to what you expect
```
Comment (freben) [inline packages/backend-tasks/README.md:35]:
```suggestion
When working with the `@backstage/backend-tasks` library you may run into your task not running immediately at startup as expected if you are using a persistent database. This is by design - the library respects the previous state and does not run the task sooner than the specified frequency. If you want to get around this, there is a table called `backstage_backend_tasks__tasks` in the applicable plugin's database will contain a record with the next run date and time. You can delete this record to get things back to what you expect.
```

Sorry to trample over your formulation, feel free to put it in your own words, but I wanted to make it clear that this is actually correct behavior. :) Unless I misunderstood the situation.

**Review by freben — COMMENTED** [review]
This actually needs a pull request too, so that the updated readme gets shipped to npm. :)

File: packages/backend-tasks/README.md (hunk 1 above)
Comment (awanlin) [reply packages/backend-tasks/README.md:35]: No, this is great and why I hopped you would look at this PR. I'll update the wording and add a changeset too 👍

**Comment by awanlin** [comment]
Ready for another review or for these changes to be approved and merged 👍

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
Thanks and happy new year!

**Pull request merged**